server/issue.go

```diff
@@ -209,6 +211,72 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 	return http.StatusOK, nil
 }
 
+func httpWorkflowCreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (int, error) {
+	if r.Method != http.MethodPost {
+		return http.StatusMethodNotAllowed,
+			errors.New("method " + r.Method + " is not allowed, must be POST")
+	}
+
+	var activationParams workflowclient.ActionActivationParams
+	if err := json.NewDecoder(r.Body).Decode(&activationParams); err != nil {
+		return http.StatusBadRequest,
+			errors.WithMessage(err, "failed to decode incoming request")
+	}
+
+	create := &struct {
+		UserId string           `json:"user_id"`
+		Fields jira.IssueFields `json:"fields"`
+	}{}
+	if err := json.NewDecoder(bytes.NewReader(activationParams.Action)).Decode(&create); err != nil {
+		return http.StatusBadRequest,
+			errors.WithMessage(err, "failed to decode incoming request specific parameters")
+	}
+
+	var client interface {
```

Comment:
I seem to remember there being a client interface we can already use?

Reply:
Ya there are multiple but they don't match. They do have this method in common so I just created an interface for what this function needs.